index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="https://index.html" method="post">
+    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <ul class="form_field">
+        <li class="form_name">
+            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name" maxlength="30" required>
+        </li>
```

Comment:
- [ ] I see you made changes but it is not the one requested by the previous reviewer. Following the project requirement, you should use [Formspree](https://formspree.io/html/) for submitting your contact inputs. Kindly make sure it is implemented correctly.
Hint: Your `form` should look like this:
```
<form action="https://formspree.io/f/{form_id}" method="post">
  <label for="email">Your Email</label>
  <input name="Email" id="email" type="email">
  <button type="submit">Submit</button>
</form>
```

Note: You should go to [Formspree](https://formspree.io/html/) and generate your unique `form_id`.

Reply:
okay good, so I'm sorry for that, but I'm following the Figma design please approve my work

Reply:
or could just reach out to me on slack before doing something here on github, try to look at this picture, so I have done, prove it, please !! 

![image](https://user-images.githubusercontent.com/74502009/177411706-486a361f-1ba6-480d-8fe3-9ea88d4de0c2.png)

Reply:
okay good, but I'm implementing this design below, so I have done it, you can just approve my work, or you could just notice me on slack to know more  before coming over hereon GitHub 👎 

![image](https://user-images.githubusercontent.com/74502009/177412044-8750a27d-8200-4f36-824f-89301a189f43.png)

Reply:
so, everything is okay, and my design looks like the design from Figma please let me pass on the follow step

Reply:
That you ask me is not that I have to do, try to run the project on your machine and then rate me, please !!
not give the conclusion without understanding the work